### ISSUE TITLE
Fasltane full_description.txt for de & fr too long

### ISSUE THREAD

**Issue by IzzySoft** [NightDreamGames/Graded#8]
Just a pointer: the limit for `full_description.txt` is 4.000 chars.  Currently, DE has 4.039 and FR 4.053 chars; both will hence be truncated.

**Comment by FauconSpartiate** [comment]
Yes I noticed this, I actually fixed it 2 days ago in 4b83dde...
But thank you 👍

**Comment by IzzySoft** [comment]
Ha! I should have checked first. But it's the first time a dev noticed and fixed it before me reporting it, and I have my monthly runs with this :see_no_evil:

Thanks a lot!

**Comment by IzzySoft** [comment]
Hrmpf… Follow up, sorry: This time it's the French changelogs. Limit there is 500 chars, the current one is at 596 at the moment. Friendly pointer, @FauconSpartiate :wink:

**Comment by FauconSpartiate** [comment]
@IzzySoft I actually removed the changelogs translations, as they were painful to make, not very precise and nobody used them.
They will only be in english now.

So that file actually doesn't exist anymore, is it still saved on your side or will it automatically get deleted too?

**Comment by IzzySoft** [comment]
non-existing files will not automatically be removed in that place I fear (old ones get removed when new ones come in, though: as my repo usually keeps the latest 3 versions anyway, there's no use to keep older changelogs). So I removed all now, except for en-US.

Err, mom… the badge for my repo is removed now from your Readme, and in #1 we concluded to remove your app here then as well. So I'll do that now. Thanks!
